### PR TITLE
#79: Added builder for ValidationFinding

### LINKED ISSUE #79: Add builder for ValidationFinding
Currently, the constructor calls for `ValidationFinding` are very unreadable. Using a builder that could be improved a lot.

### PR DESCRIPTION
**Pull request by jakobbraun**
Closes #79

### CODE REVIEW

File: src/main/java/com/exasol/projectkeeper/ValidationFinding.java [hunk 1]
```diff
@@ -69,4 +59,44 @@ public boolean hasFix() {
...
+        private Fix fix;
+
+        public Builder(final String message) {
+            this.message = message;
+        }
+
+        /**
+         * Add a fix to the finding (optional).
+         * <p>
+         * You can only add one fix. IF you call this method multiple times the fix will be overwritten.
```
Comment (morazow) [inline src/main/java/com/exasol/projectkeeper/ValidationFinding.java:87]:
```suggestion
         * You can only add one fix. If you call this method multiple times the fix will be overwritten.
```

File: src/main/java/com/exasol/projectkeeper/ValidationFinding.java [hunk 2]
```diff
@@ -69,4 +59,44 @@ public boolean hasFix() {
...
+         * </p>
+         * 
+         * @param fix function that fixes the finding
+         * @return self for fluent programming
+         */
+        public Builder andFix(final Fix fix) {
```
Comment (morazow) [inline src/main/java/com/exasol/projectkeeper/ValidationFinding.java:93]: Is it intended to be single fix?

Another option would be to take a list of fixes and apply them in order.

File: src/main/java/com/exasol/projectkeeper/ValidationFinding.java [hunk 3]
```diff
@@ -69,4 +59,44 @@ public boolean hasFix() {
...
+    public static class Builder {
+        private final String message;
+        private Fix fix;
+
+        public Builder(final String message) {
+            this.message = message;
+        }
+
+        /**
+         * Add a fix to the finding (optional).
```
Comment (morazow) [inline src/main/java/com/exasol/projectkeeper/ValidationFinding.java:85]:
```suggestion
         * Add an optional fix to the finding.
```

**Review by morazow — CHANGES_REQUESTED** [review]

File: src/main/java/com/exasol/projectkeeper/ValidationFinding.java (hunk 2 above)
Comment (jakobbraun) [reply src/main/java/com/exasol/projectkeeper/ValidationFinding.java:93]: yes. There is always exactly one  fix per finding.

**Review by morazow — APPROVED** [review]

**Pull request merged**